challenge/dns01/precheck.go

```diff
@@ -44,10 +44,15 @@ func RecursiveNSsPropagationRequirement() ChallengeOption {
 	}
 }
 
-func PropagationWaitOnly(wait time.Duration) ChallengeOption {
+func PropagationWait(wait time.Duration, only bool) ChallengeOption {
```

Comment:
The name `only` does not really convey a meaning (as did the previous function name, but I didn't catch that in #2266).

Something like `onlySleep` or `noCheck` would be more descriptive, but is maybe too close to the implementation? Maybe this function just needs a bit of documentation.

Reply:
Sounds good :+1: